.gitignore

```diff
@@ -23,6 +23,7 @@ website/node_modules
 *.iml
 *.test
 *.iml
+tags*
```

Comment:
this ignores ctags generated tag files which (many?) developer's have their vim to automatically generate in cloned directories.